js-miniapp-sdk/src/miniapp-error-type.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * This class is a representation of an error sent from MiniApp mobile SDK
+ */
+export class MiniAppError extends Error {
```

Comment:
Any custom classes and types that you want to be visible to users of the SDK need to be imported to and exported from `js-miniapp-sdk/src/index.ts`.